src/components/HURUmap/Map/index.js

```diff
@@ -78,7 +81,11 @@ function Map({
       };
     }
     setSelectedBoundary(selectedBound);
-  }, [geometries, geography]);
+  }, [geometries, geography, getSelectedBoundary]);
+
+  const locationCodes = Object.keys(featuredGeographies).reduce((acc, v) => {
+    return acc.concat(featuredGeographies[v]);
+  }, []);
```

Comment:
Wouldn't this be computed on every re-render even when `featuredGeographies` haven't changed?

Reply:
@kilemensi right, I will move it at a higher level